### PR TITLE
feat: add subscriber video stats to analytics

### PR DESCRIPTION
**Pull request by eswarclynn**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/hms-video-store/src/utils/constants.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ export const PUBLISH_STATS_PUSH_INTERVAL = 300;
 export const SUBSCRIBE_STATS_SAMPLE_WINDOW = 10;
 export const SUBSCRIBE_STATS_PUSH_INTERVAL = 60;
 
+export const JAVA_INTEGER_MAX_VALUE = 2147483647;
```
Comment (raviteja83) [inline packages/hms-video-store/src/utils/constants.ts:37]: JAVA?

Comment (raviteja83) [reply packages/hms-video-store/src/utils/constants.ts:37]: Math.pow(2,31)-1

File: packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts [hunk 2]
```diff
@@ -75,6 +82,25 @@ export class SubscribeStatsAnalytics extends BaseStatsAnalytics {
...
+  private calculateAvSyncForStat(trackStats: HMSTrackStats, hmsStats: HMSWebrtcStats) {
+    if (!trackStats.peerID || !(trackStats.kind === 'video')) {
+      return;
+    }
+    const peer = this.store.getPeerById(trackStats.peerID);
+    const audioTrack = peer?.audioTrack;
+    const videoTrack = peer?.videoTrack;
+    const areBothTracksEnabled = audioTrack && videoTrack && audioTrack.enabled && videoTrack.enabled;
+    if (!areBothTracksEnabled) {
+      return JAVA_INTEGER_MAX_VALUE;
```
Comment (raviteja83) [inline packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:96]: we could use Number.POSITIVE_INFINITY or Number.MAX_SAFE_INTEGER. does not work?

Comment (eswarclynn) [reply packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:96]: The value needs to be same across clients, the JS value is different from the value Java uses.

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**